hardware_interface/include/skytraq_binary.h

```diff
@@ -5,7 +5,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#define GPS_UART_TIMEOUT_MS 1000 * portTICK_PERIOD_MS
+#define GPS_UART_TIMEOUT_MS 2000 * portTICK_PERIOD_MS
```

Comment:
That's a very long time. Why so long?

Reply:
There was one command that took like 1.5s